src/pages/Swap/LimitOrderBox/components/LimitOrderForm/LimitOrderForm.tsx

```diff
@@ -476,6 +477,7 @@ export function LimitOrderForm({ account, provider, chainId }: LimitOrderFormPro
               isFallbackFiatValue={isFallbackFiatValueInput}
               showNativeCurrency={false}
               currencyOmitList={[buyTokenAmount.currency.address!]}
+              currencySelectWrapper={CurrencySelectTooltip}
```

Comment:
what is the use of passing it here at props? I think we should just include it in component since its always used...

Reply:
We can't show this tooltip in Swap and Bridge boxes, so it's being passed as a prop.